### DISCUSSION TITLE
API question - Latest snapshot for label

### DISCUSSION THREAD

**Discussion by DrSpaldo** [blakeblackshear/frigate#6722] [Ask A Question]
Hi, I am setting up an automation and would like to display the latest person snapshot. I have found the following api:

```GET /api/<camera_name>/<label>/snapshot.jpg```

It looks like I can do "any" for the label but not the camera. I would like the label to be "person" but the camera to be "any". Is this possible? am I missing something?

Thanks

**Comment by NickM-27** [comment]
Currently there is no single call for this, you'd need to call events to find the latest person event and then get the snapshot for that event id.

**Reply by DrSpaldo** [reply]
> Currently there is no single call for this, you'd need to call events to find the latest person event and then get the snapshot for that event id.

That's a shame, I would have thought it would be logical to have that if label had that option. How does one 'call events to find the latest person event'?

**Reply by NickM-27** [reply]
Most APIs are added out of need, there was no need to get the latest object event for any camera so it wasn't added.

You can get the latest person event using `/api/events?labels=person&limit=1`

**Reply by DrSpaldo** [reply]
Alright, so I run that command, then get the massive output, get Home Assistant to convert the camera output into the camera word, then put that variable into the original command
